example/road-traffic/road-traffic-demo/pom.xml

```diff
@@ -83,6 +83,18 @@
                                     href='http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3'
                                     &gt;Open Government Licence&lt;/a&gt;.
                                 </gaffer.properties.app.description>
+                                <gaffer.properties.app.logo.link>
+                                    https://github.com/gchq/Gaffer
+                                </gaffer.properties.app.logo.link>
+                                <gaffer.properties.app.logo.src>
+                                    images/iconCircle.png
```

Comment:
Please could you rename the iconCircle.png file to logo.png and move it into src/main/webapp/images/